packages/edit-site/src/components/layout/index.js

```diff
@@ -245,39 +250,73 @@ export default function Layout( { onError } ) {
 
 				{ showCanvas && (
 					<div
-						className="edit-site-layout__canvas-container"
+						className={ classnames(
+							'edit-site-layout__canvas-container',
+							{
+								'is-resizing': isResizing,
+							}
+						) }
 						style={ {
 							paddingTop: showFrame ? canvasPadding : 0,
 							paddingBottom: showFrame ? canvasPadding : 0,
 						} }
 					>
-						{ canvasResizer }
-						{ !! canvasSize.width && (
-							<motion.div
-								initial={ false }
-								layout="position"
-								className="edit-site-layout__canvas"
-								transition={ {
-									type: 'tween',
-									duration: disableMotion
-										? 0
-										: ANIMATION_DURATION,
-									ease: 'easeOut',
-								} }
-							>
+						{ canvasContainerResizer }
+						<ResizableBox
+							size={ {
+								height: '100%',
+								width:
+									! isMobileViewport &&
+									isEditorPage &&
+									canvasMode === 'view'
+										? forcedWidth ?? '100%'
+										: '100%',
+							} }
+							className="edit-site-layout__resize-handle-container"
+							minWidth="100"
+							maxWidth={ canvasContainerSize.width }
+							enable={ {
+								left:
+									! isMobileViewport &&
+									isEditorPage &&
+									canvasMode === 'view',
+							} }
+							onResizeStop={ ( _, _2, _3, delta ) => {
+								setForcedWidth(
+									parseInt(
+										( forcedWidth ?? canvasSize.width ) +
```

Comment:
I'm noticing something strange. While I resize, the mouse moves much faster than what's being resized (I think because it is centered). Maybe we can fix that by multiplying the delta by two, given that as it is centered, we are resizing from the right and left.